### ISSUE TITLE
Pareto distribution

### ISSUE THREAD

**Issue by curran** [d3/d3-random#1]
It might be nice to add a [Pareto distribution](https://en.wikipedia.org/wiki/Pareto_distribution) generator. 

Here is an implementation, using math from [http://www.cse.wustl.edu/~jain/books/ftp/ch5f_slides.pdf](http://www.cse.wustl.edu/~jain/books/ftp/ch5f_slides.pdf), page 32:

``` javascript
function pareto(alpha){
  return function(){
    return 1 / Math.pow(Math.random(), 1 / alpha);
  };
}
```

Also, here's an [example that uses this implementation](http://bl.ocks.org/curran/11301130). You can see visually that the distribution looks correct, but I could not wrap my head around how to make a proper test for this. Unfortunately I cannot offer a complete implementation with tests, but I just thought I'd put this idea/feature request up here.

**Comment by svanschooten** [comment]
See #9 .
We're adding more distributions in [our fork](https://github.com/delftswa2016/d3-random)

**Comment by curran** [comment]
Nice!

**Comment by anooprh** [comment]
@svanschooten 
Is there any reason these why these are not merged/not created a PR to this repo ? Especially `bernoulli` and `geometric` distributions are very useful.

**Comment by svanschooten** [comment]
@anooprh According to #9 it has been merged?

**Comment by mbostock** [comment]
Fixed in #14 d970d8f39d5ec9d3e33b334a0e80bc7e8e42d7a8.
